src/.eslintrc.js

```diff
@@ -63,7 +63,7 @@ module.exports = {
           },
           {
             pattern:
-              "(analytics|components|constants|context|gql|hoc|hooks|pages|types|utils)/**",
+              "{analytics,components,constants,context,gql,hoc,hooks,pages,test_utils,types,utils}{,/**}",
```

Comment:
I still think there is something up with this statement. 
![image](https://user-images.githubusercontent.com/4605522/167190937-5b81fb65-b10c-4dd8-9a95-b5ef86255f47.png)
```ts
import { useState } from "react";
import { useQuery } from "@apollo/client";
import styled from "@emotion/styled";
import Tooltip from "@leafygreen-ui/tooltip";
import { useLocation } from "react-router";
import { useSpawnAnalytics } from "analytics";
import { ConditionalWrapper } from "components/ConditionalWrapper";
import { PlusButton } from "components/Spawn";
import { size } from "constants/tokens";
import {
  MyHostsQuery,
  MyHostsQueryVariables,
  GetSpruceConfigQuery,
  GetSpruceConfigQueryVariables,
} from "gql/generated/types";
import { GET_MY_HOSTS, GET_SPRUCE_CONFIG } from "gql/queries";
import { SpawnHostModal } from "pages/spawn/spawnHost/index";
import { HostStatus } from "types/host";
import { queryString } from "utils";
```